native/styled-components-native-test.tsx

```diff
@@ -0,0 +1,20 @@
+/// <reference path="index.d.ts" />
```

Comment:
If you follow my above advice about moving the module out of the `declare module` block, you should just be able to use imports instead of a `/// <reference ... />` comment.